docs/content/resource-authors-guide.md

```diff
@@ -110,10 +110,11 @@ your code as possible.
 ## Task
 
 The
-[`resource.Task`](https://godoc.org/github.com/asteris-llc/converge/resource#Task)
-interface is what you will implement to have converge run your `Check` and
-`Apply` methods.  Your `resource.Task` implementation is also what's used to
-define lookup methods from within converge.  An example of a
+[`resource.Task`](https://godoc.org/github.com/asteris-llc/converge/resource#Task) interface
+is what you will implement to have converge run your `Check` and `Apply`
+methods.  The `export` and `re-export-as` tags in your `resource.Task`
+implementation are used to define lookup methods from within converge.  An
+example of a resource is:
```

Comment:
Maybe change to "We will use the Shell resource in the following examples:" or something like that. This will also address my comment below on mentioning Shell earlier (line 152)